### ISSUE TITLE
Issue: Image pasting does not work

### ISSUE THREAD

**Issue by zozzzC** [Pseudonium/Obsidian_to_Anki#567]
Hi, I'm using custom regex here but I had no issues with images in the past until now. I've tried using downloaded png images as well as images I have copy pasted online, but either way images are no longer showing up on my deck. I didn't used to have this issue before. 

(The card in my vaults VS On my Anki)
<img width="738" alt="image" src="https://github.com/ObsidianToAnki/Obsidian_to_Anki/assets/104532983/251307b8-1c10-4d3b-949e-ca6d4e2d90ad">

<img width="738" alt="image" src="https://github.com/ObsidianToAnki/Obsidian_to_Anki/assets/104532983/32cf9ec4-29b2-4c7a-9977-4e1e581f270a">

**Comment by advikchau** [comment]
Same issue.

**Comment by advikchau** [comment]
This seems to happen when using Google Drive to store/sync the vault. Works with iCloud.

**Comment by advikchau** [comment]
Yep, this might be a google drive issue. The images load if I choose a vault that is stored locally (on mac), then synced with Google Drive (by manually selecting the folder to sync)—as opposed to just accessing from the Google Drive tab/directory directly.

**Comment by zozzzC** [comment]
I'm using Obsidian Git and not Google Drive, so I'm not too sure about the Google Drive thing.

**Comment by advikchau** [comment]
Ah, I see. Maybe try GDrive or iCloud instead? Works for me currently.

**Comment by zozzzC** [comment]
After checking media in Anki, it seems that Anki cannot find the file referenced by the card.

**Comment by zozzzC** [comment]
An update on this: I think there's some issue with how Obsidian Git either syncs or clones repos, since I tried the plugin on a new vault and images worked just fine. I cloned my repo a few months back and that is when I started getting this issue. I might just move everything to a new vault and see if that fixes things.

**Comment by zozzzC** [comment]
Alright I think I know what the issue is. #349 solves it--for some reason I keep getting the error that AnkiConnect needs to update on the console. So for some reason images are not exporting correctly. I just did what @serpro69 did and manually copy pasted my media files in Obsidian into Anki.

**Comment by LukazLu** [comment]
Same problem as [Obsidian-Anki-Sync](https://github.com/debanjandhar12/Obsidian-Anki-Sync) plugin.

[Images Not Displaying Correctly: Unable to Sync with Anki's Media Folder](https://github.com/debanjandhar12/Obsidian-Anki-Sync/issues/25)

**Comment by liujiaqi222** [comment]
Once I disabled the wiki links, the images began displaying correctly
